uwblueprint/planet-read
# Frontend/view profile

## Notion ticket link



[View profile + logout drop down from nav bar](https://www.notion.so/uwblueprintexecs/View-profile-logout-drop-down-from-nav-bar-7efd92729fb147d0b935e6b3d81b0344)



## Implementation description

- Added user profile modal using Chakra UI Modal
- Profile and sign out links are text with `onClick` for consistent appearance
- Icons from `react-icons/bi` most closely matched those in Figma



## Steps to test

1. Sign in as `planetread+angelamerkel@uwblueprint.org`
2. Click on user icon in nav bar
3. Modal should say Angela Merkel Admin with option to sign out
4. Click away from the modal to close it
5. Reopen the modal and sign out
6. Sign in as `planetread+kevindebruyne@uwblueprint.org`
7. Click on user icon in nav bar
8. Modal should say Kevin De Bruyne Translator & Reviewer with options view profile and sign out
9. Click on profile to be redirected to user page
10. Go back
11. Reopen the modal and sign out
12. Sign in as `planetread+carlsagan@uwblueprint.org` (for me Carl is broken so I used `planetread+miroslavklose@uwblueprint.org` but removed his reviewer status using mysql)
13. Click on user icon in nav bar
14. Modal should say Carl Sagan Translator (or Miroslav Klose Translator) with options view profile and sign out
15. Click on profile to be redirected to user page
16. Go back
17. Reopen the modal and sign out



## What should reviewers focus on?

- Do links work
- Is code readable enough
- Is appearance good

## Checklist
